src/main/java/org/wildfly/security/http/util/SecurityProviderServerMechanismFactory.java

```diff
@@ -83,6 +85,9 @@ public SecurityProviderServerMechanismFactory() {
                     }
                 }
             }
+            if ( ! containFactory) {
+                log.noHttpServerAuthenticationMechanismFactoryProvidedByProvider(current);
```

Comment:
I am thinking, does it make sense to start server in such case at all? Isn't it better to throw exception here? I can't think of example when this could be intentional - user setting explicitly special server mechanism factory without any mechanism authentication.